.github/ISSUE_TEMPLATE/10_bug_report.yml

```diff
@@ -0,0 +1,45 @@
+name: Bug report
+description: Create a bug report.
+labels:
+  - 'Bug'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report correctly. Do NOT use GitHub to post any questions or support requests! They will be closed immediately and ignored.
+
+        The GDAL project is made of contributions from various individuals and organizations, each with their own focus. The issue you are facing is not necessarily in the priority list of those contributors and consequently there is no guarantee that it will be addressed in a timely manner. If this bug report or feature request is high-priority for you, and you cannot address it yourself, we suggest engaging a GDAL developer or support organisation and financially sponsoring a fix.
+
+  - type: textarea
+    id: what
+    attributes:
+      label: What is the bug?
+      description: |
+        If you think there is an issue with coordinate order in GDAL 3.0, then it is likely an intended behavior. See https://github.com/OSGeo/gdal/issues/1974 for more explanations.
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce the issue
+      description: |
+        Steps, code extract and/or sample datasets to reproduce the behavior.
+    validations:
+      required: true
+
+  - type: textarea
+    id: about-info
+    attributes:
+      label: Versions
+      description: |
+        Please indicate the operating system (e.g "Windows 11", or "Linux Ubuntu 22.04") and the GDAL version, for example by pastining the output of ``gdalinfo --version``.
+    validations:
```

Comment:
"pastining" -> pasting
Explain that the GDAL project supports only the current version https://gdal.org/download.html#current-release so it should be used for testing. Or at least the newest easily available version.

Reply:
fixed. Also added your hints